libevmasm/SemanticInformation.cpp

```diff
@@ -72,6 +87,7 @@ std::vector<SemanticInformation::Operation> SemanticInformation::readWriteOperat
 	case Instruction::LOG4:
 	{
 		assertThrow(storage(_instruction) == Effect::None, OptimizerException, "");
+		assertThrow(transientStorage(_instruction) == Effect::None, OptimizerException, "");
 		assertThrow(memory(_instruction) == Effect::Read, OptimizerException, "");
```

Comment:
It would be nice to keep the orders of these `assertThrow`s the same as other cases:

1. `memory`
2. `storage`
3. `transientStorage`